jupyter/jsl_installation_script_ubuntu.sh

```diff
@@ -188,7 +212,13 @@ then
         echo "Reading Combined License json from $comb_json_path ..."
         export_json "$comb_json_path"
     fi
-    
+
+    if ! [ "$myjsl_access_token" == false ]
```

Comment:
Why do we need this if we export env variables above?

Reply:
It's for convenience, because for running this bash user may just -t flag same as install and don't use above path flags